### DISCUSSION TITLE
Cant save email configuration - "createdAt" is not allowed

### DISCUSSION THREAD

**Discussion by Hedronmx** [Budibase/budibase#6668] [Help] [answered]
Hey guys! Loving this piece of software.

Just a problem, i cant seem to save the email configuration. I get the following error

Failed to save email settings, reason: Invalid body - "createdAt" is not allowed

Any ideas? Budibase - 1.0.218

**Comment by mike12345567** [comment, marked as answer]
Hi @Hedronmx I believe this is an issue with the API validators, I'll raise an issue for this.

Edit: #6681
